core/templates/dev/head/pages/exploration_editor/EditorModeServiceSpec.js

```diff
@@ -16,25 +16,25 @@
  * @fileoverview Unit tests for the EditorModeService.js
  */
 describe('Editor Mode Service', function() {
-  var service;
+  var srv;
```

Comment:
Still no. The variable name takes the initials of the service that is being tested.